### ISSUE TITLE
Touch tracking on iOS cannot register an initial movement less than many pixels

### ISSUE THREAD

**Issue by celyk** [godotengine/godot#90614]
### Tested versions

v4.2.1.stable.official [b09f793f5]

### System information

iPadOS 17.4, iPad 9th gen, Apple Pencil 1st gen

### Issue description

This is the iOS analogue of #84138. I'm unable to get touch events within about 15px from the initial `InputEventScreenTouch`, causing small jumps while attempting to drag slowly for fine control.

In addition to a Godot project MRP, I've included a Swift Playground displaying the same problem. The Apple pencil doesn't have this problem.
![touch vs apple pencil](https://github.com/godotengine/godot/assets/50609684/868ab320-33b6-4ac2-9740-70507ca95d1f)

https://github.com/godotengine/godot/assets/50609684/5dc18dc0-1d42-43a2-a550-312a47654d4f

### Steps to reproduce

Do one:
- Export the Godot project for iOS
- Run the Swift Playground directly in the app for iPadOS

You should see a small gap between the initial touch point (red) and the drag points (blue). I have offset all points by 100px for visibility.

### Minimal reproduction project (MRP)

[godot-display-touch-events.zip](https://github.com/godotengine/godot/files/14965848/godot-display-touch-events.zip)
[swift-display-touch-events.zip](https://github.com/godotengine/godot/files/14965780/swift-display-touch-events.zip)

**Comment by kraftwer1** [comment]
Yes, I can confirm this. And it does not happen on Android. Since my game requires quite high precision, it would be very nice if this could be fixed.
